servers/audio_server.cpp

```diff
@@ -404,8 +404,22 @@ void AudioServer::_mix_step() {
 			buf[i] = playback->lookahead[i];
 		}
 
-		// Mix the audio stream.
-		unsigned int mixed_frames = playback->stream_playback->mix(&buf[LOOKAHEAD_BUFFER_SIZE], playback->pitch_scale.get(), buffer_size);
+		unsigned int mixed_frames = 0;
```

Comment:
`uint64_t scheduled_start_frame` and `unsigned int mixed_frames = 0;` should be the same type like uint64_t.

Reply:
I think these should be different types given that `mixed_frames` is limited by `buffer_size`, which is `uint32_t`. Conversely, `scheduled_start_frame` is uncapped and will overflow at ~27 hours with the default mix rate with 32 bits, so it should be `uint64_t`.

For better type consistency, I've changed `silence_frames` back to `unsigned int` with an explicit cast.